modules/@apostrophecms/ui/ui/apos/components/AposFile.vue

```diff
@@ -160,16 +160,19 @@ export default {
 };
 </script>
 <style scoped lang='scss'>
+
   .apos-file-dropzone {
     @include apos-button-reset();
```

Comment:
You removed the parenthesis after the `apos-button-reset` mixin but not in all cases, what is the difference between both?

Reply:
Again, I was mostly cutting and pasting. I'm not sure how the lose of parenthesis crept in - no arguments, so they aren't needed, but it should be consistent. Thanks